### PR TITLE
add applications files

### PR DESCRIPTION
**Pull request by hoangtm1601**
add applications files

### CODE REVIEW

File: resources/views/clients/applications/candidates/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+@endsection
+
+
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/index.blade.php]: xóa dòng trắng dư

File: resources/views/clients/applications/candidates/job_element_square.blade.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                        <p>
+                            <a href="#"><h4>{{ $job['company_name'] }}</h4></a>
+                        </p>
+                        <ul>
+                            <li>
+                                <i class="fa fa-usd"></i>&nbsp;
+                                {{ number_format($job['job']->salary_min) . ' - ' . number_format($job['job']->salary_max) }}
+                            </li>
+                            <li><i class="fa fa-map-marker"></i>&nbsp; {{ $job['job']->locationJobs->name }}</li>
+                            <li>{{__('Date applied: ') . $job['applied_date'] }}</li>
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/job_element_square.blade.php]: convention

File: resources/views/clients/applications/candidates/job_element_square.blade.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                    <div class="jp_job_post_right_btn_wrapper jp_job_post_grid_right_btn_wrapper">
+                        <ul class="pull-right">
+                            <li></li>
+                            <li><a href="#">{{ $job['job']->jobTypeJobs->name }}</a></li>
+                            <li><a href="{{ route('jobs.detail', ['id' => $job['job']->id]) }}">Detail</a></li>
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/job_element_square.blade.php]: i18n

File: resources/views/clients/layouts/nav.blade.php [hunk 4]
```diff
@@ -1,93 +1,88 @@
...
-                                <a href="javascript:void(0)"
-                                   onclick="logout('{{ __('conten_logout') }}', '{{ __('cancel') }}', '{{ __('ok') }}');">
-                                    {{ __('logout') }}
-                                    <i class="glyphicon glyphicon-log-out"></i>
-                                </a>
-                            </li>
+                <div class="col-lg-2 col-md-2 col-sm-12 col-xs-12 hidden-sm hidden-xs">
+                    <div class="jp_navi_right_btn_wrapper">
+                        <ul>
+                            <li><a href="{{ route('jobs.create') }}"><i class="fa fa-plus-circle"></i>&nbsp; Post a job</a></li>
```
Comment (BunnyPi04) [inline resources/views/clients/layouts/nav.blade.php]: i18n

File: resources/views/clients/layouts/top_header.blade.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                                </li>
+                            @elseauth
+                        </ul>
+                        @endauth
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (BunnyPi04) [inline resources/views/clients/layouts/top_header.blade.php]: thiếu dòng trắng cuối file

**Comment by hoangtm1601** [comment]
ready

File: app/Http/Controllers/ApplyJobController.php [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+            if ($recentlyAddedApplication) {
+                return redirect()->route('jobs.detail', ['id' => $jobId])
+                    ->with('status',__('Job applied'));
+            } else {
+                return redirect()->route('jobs.detail', ['id' => $jobId])
+                    ->with('status',__('Apply failed'));
+            }
+        } else {
+            return redirect()->route('jobs.detail', ['id' => $jobId])
+                ->with('status',__('You have applied this job'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/ApplyJobController.php]: convention

**Comment by hoangtm1601** [comment]
ready

File: resources/views/clients/layouts/nav_not_auth.blade.php [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+
```
Comment (BunnyPi04) [inline resources/views/clients/layouts/nav_not_auth.blade.php]: xóa dòng trắng dư

File: resources/views/clients/layouts/top_header.blade.php [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+                        <ul>
+                            <li>
+                                {!! Form::select('change_lang',[
+                                        'vi' => 'Vi',
+                                        'en' => 'En',
+                                     ],
+                                     session()->has('lang') ? session()->get('lang') : 'en', [
+                                        'class' => 'selectpicker form-control mt-10',
+                                        'id' => 'change_lang',
+                                        'onChange' => 'changeLang(\'' .url(''). '\')',
```
Comment (BunnyPi04) [inline resources/views/clients/layouts/top_header.blade.php]: convention

File: resources/views/clients/layouts/nav_not_auth.blade.php [hunk 9]
```diff
@@ -0,0 +1,60 @@
+<div class="gc_main_menu_wrapper">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-lg-2 col-md-2 col-sm-12 col-xs-12 hidden-xs hidden-sm full_width">
+                <div class="gc_header_wrapper">
+                    <div class="gc_logo">
+                        <a href="/">
```
Comment (BunnyPi04) [inline resources/views/clients/layouts/nav_not_auth.blade.php]: ko dùng đường dẫn tương đối

File: resources/views/clients/applications/candidates/job_element_square.blade.php [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+                            <li></li>
+                            <li><a href="#">{{ $job['job']->jobTypeJobs->name }}</a></li>
+                            <li><a href="{{ route('jobs.detail', ['id' => $job['job']->id]) }}">{{ __('Detail') }}</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="jp_job_post_keyword_wrapper">
+            @foreach($job['skills'] as $skill)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/job_element_square.blade.php]: convention

File: resources/views/clients/applications/candidates/index.blade.php [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+                                    <div class="row">
+                                        @foreach($jobs as $job)
+                                            @include('clients.applications.candidates.job_element_square')
+                                        @endforeach
+                                        @include('clients.home.partials.pagination')
+                                    </div>
+                                </div>
+                                <div id="list" class="tab-pane fade">
+                                    <div class="row">
+                                        @foreach($jobs as $job)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/index.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_25-khoi-lenh-10

File: resources/views/clients/applications/candidates/index.blade.php [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                            <div class="tab-content">
+                                <div id="grid" class="tab-pane fade in active">
+                                    <div class="row">
+                                        @foreach($jobs as $job)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/index.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_25-khoi-lenh-10

File: resources/views/clients/applications/candidates/job_element_retangle.blade.php [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+                            <li></li>
+                            <li><a href="#">{{ $job['job']->jobTypeJobs->name }}</a></li>
+                            <li><a href="{{ route('jobs.detail', ['id' => $job['job']->id]) }}">{{ __('Detail') }}</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="jp_job_post_keyword_wrapper">
+            @foreach($job['skills'] as $skill)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/candidates/job_element_retangle.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_25-khoi-lenh-10

File: resources/views/clients/applications/form.blade.php [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+            'placeholder' => __('What skills, work projects or achievements make you a strong candidate? (Recommended)'),
+        ]) }}
+        {!! $errors->first('self_introduction', '<span class="red">:message</span>') !!}
+    </div>
+</div>
+<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+    <div class="jp_adp_choose_resume_bottom_btn_post">
+        <ul>
+            <li>
+                @if($canApply)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/form.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_25-khoi-lenh-10

File: resources/views/clients/applications/job_element.blade.php [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+                            <li></li>
+                            <li><a href="#">{{ $job['job']->jobTypeJobs->name }}</a></li>
+                            <li><a href="{{ route('jobs.detail', ['id' => $job['job']->id]) }}">{{ __('Detail') }}</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="jp_job_post_keyword_wrapper">
+            @foreach($job['skills'] as $skill)
```
Comment (BunnyPi04) [inline resources/views/clients/applications/job_element.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_25-khoi-lenh-10

**Pull request merged**